### DISCUSSION TITLE
Failed to enable unit: Unit file cri-docker.service does not exist.

### DISCUSSION THREAD

**Discussion by CS8867** [Mirantis/cri-dockerd#193] [Q&A]
Followed all the steps mentioned in the documentation, the cri-docker.service file also seems to be getting created in /etc/systemd/system. However it's not getting recognized as a unit file. Any help would be greatly appreciated

![cri-docker-error](https://github.com/Mirantis/cri-dockerd/assets/60603915/46840b96-183c-4a3c-b5c8-5259a7807747)

The process followed is illustrated above.

**Comment by neersighted** [comment]
See https://github.com/Mirantis/cri-dockerd/issues/146; you really should prefer the packages if you can. Commenting with https://github.com/Mirantis/cri-dockerd/issues/154 on why you're here/using cri-dockerd could also be helpful (as this is really meant to be a repository of source code, and the README instructions are aimed at developers of cri-dockerd); who the end-user of **this repository**/cri-dockerd without a packager/distributor is not really known yet.

**Comment by afbjorklund** [comment]
The problem is the `cp -a`, which destroys the Linux context

It really needs to be replaced with `install`, in the docs

**Reply by neersighted** [reply]
Agreed; it could help with some of the bleeding even if the bigger issue is #154 -- would you mind sending a PR?

**Reply by afbjorklund** [reply]
Sure, it could need more fixes though. And building from source should probably come after installing from binary?

**Reply by neersighted** [reply]
That goes back to #154 and trying to identify who end users coming here are/how to help them, and if this is the right place for them I think...

Putting a band-aid on it ought to be better than the status quo while we figure out the bigger picture I think.
